docs/quickstarts/nextjs.mdx

```diff
@@ -102,7 +102,7 @@ description: Add authentication and user management to your Next.js app with Cle
   - [`<SignedIn>`](/docs/components/control/signed-in): Children of this component can only be seen while **signed in**.
   - [`<SignedOut>`](/docs/components/control/signed-out): Children of this component can only be seen while **signed out**.
   - [`<UserButton />`](/docs/components/user/user-button): A prebuilt component that comes styled out of the box to show the avatar from the account the user is signed in with.
-  - [`<SignInButton />`](/docs/components/unstyled/sign-in-button): An unstyled component that links to the sign-in page or displays the sign-in modal.
+  - [`<SignInButton />`](/docs/components/unstyled/sign-in-button): An unstyled component that links to the sign-in page. For this example, it links to the [Account Portal sign-in page.](/docs/customization/account-portal/overview)
```

Comment:
Maybe say something among the lines of:
Since we have not specified any props or env variables for the sign in url, you will automatically be redirected to AP.

Reply:
done! [43b3584](https://github.com/clerk/clerk-docs/pull/1533/commits/43b35844e50fb913012a87226f4cfe03b40dc2e6)